packages/sdk/src/evm/functions/getBalance.ts

```diff
@@ -0,0 +1,23 @@
+import { getSignerAndProvider } from "../core/classes/rpc-connection-handler";
+import { NetworkInput } from "../core/types";
+import { AddressOrEns } from "../schema/shared";
+import type { IERC20 } from "@thirdweb-dev/contracts-js";
+import IERC20Abi from "@thirdweb-dev/contracts-js/dist/abis/IERC20.json";
+import { Contract, BigNumber } from "ethers";
+
+export type GetBalanceParams = {
+  network: NetworkInput;
+  contractAddress: AddressOrEns;
+  walletAddress: AddressOrEns;
+};
+
+export async function getBalance(params: GetBalanceParams): Promise<BigNumber> {
+  const [, provider] = getSignerAndProvider(params.network, {});
+  const contract = new Contract(
+    params.contractAddress,
+    IERC20Abi,
```

Comment:
I know this is a quick version to just play with it but I think we'll get this ABI (or even the pure function to call itself) from the `Extension` later

Reply:
yes would be much preferable